src/sidebar/img/glyph-pin-pinned-12.svg

```diff
@@ -1,24 +1,6 @@
-<?xml version="1.0"?>
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Are all of these new icons licensed with MPL2/in the Firefox source code?

Reply:
They are from here: https://design.firefox.com/icons/viewer/